app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TweetsController < ApplicationController
+
+  def create
+    user_id = params[:user_id]
+    body = params[:body]
+    quote = params[:quote]
+    retweet = params[:retweet]
+  
+    @tweet = Tweet.new(user_id: user_id, body: body, quote: quote, retweet: retweet)
+  
+    respond_to do |format|
+      if @tweet.save
+        format.json { render json: { tweet: @tweet }, status: :created }
```

Comment:
Your test failed in checking the status, the test is checking 204 no content and you are sending 201 created